src/main/kotlin/graphics/scenery/controls/behaviours/VRPress.kt

```diff
@@ -56,7 +67,16 @@ open class VRPress(
      * @param y invalid - residue from parent behavior. Use [controllerSpatial] instead.
      */
     override fun drag(x: Int, y: Int) {
-        selected.forEach { it.getAttributeOrNull(Pressable::class.java)?.onHold?.invoke() }
+        selected.forEach {
+            it.getAttributeOrNull(Pressable::class.java)?.let { pressable ->
```

Comment:
This should be shortened by using `ifHasAttribute()`

Reply:
its not really shorter but I changed it